go/api/request_routing_config.go

```diff
@@ -0,0 +1,189 @@
+package api
```

Comment:
Add a licence header.
Does go have a linter that validates headers?

Reply:
Added.